### ISSUE TITLE
When pressing ctrl + d to create a bookmark, the menu thing doesn't show up.

### ISSUE THREAD

**Issue by Grooty12** [xarantolus/userchrome.css#1]
Title. It's not a serious issue, but just looking a bit weird.

**Comment by xarantolus** [comment]
That's true, it looks weird. I tried a few things to fix it, but it's really hard to style an element when a descendant element changed, so I'm not sure how this can be resolved
